libraries/Microsoft.Bot.Schema/Teams/BatchFailedEntry.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Newtonsoft.Json;
+
+namespace Microsoft.Bot.Schema.Teams
+{
+    /// <summary>
+    /// Specifies the failed entry with its id and error.
+    /// </summary>
+    public class BatchFailedEntry
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="BatchFailedEntry"/> class.
+        /// </summary>
+        public BatchFailedEntry()
+        {
+        }
+
+        /// <summary>
+        /// Gets or sets the id of the failed entry.
+        /// </summary>
+        /// <value>The id of the failed entry.</value>
+        [JsonProperty(PropertyName = "id")]
+        public string Id { get; set; }
```

Comment:
in the response we are returning property name will be called "entryId"

Reply:
Changed. Thanks!